rpm/qtbase.spec

```diff
@@ -908,5 +912,6 @@ ln -s %{_sysconfdir}/xdg/qtchooser/5.conf %{buildroot}%{_sysconfdir}/xdg/qtchoos
 %files -n qt5-default
 %defattr(-,root,root,-)
 %{_sysconfdir}/xdg/qtchooser/default.conf
+%{_sysconfdir}/profile.d/qt_bin_path.sh
```

Comment:
qt5- default is not installed by default and doing that pulls in qtchooser which kind of conflicts with the PATH thing

Have been hoping we could be getting rid of qtchooser as upstream has been reluctant on having that migrated to qt6. (was that dead already? not sure)

Reply:
Oh right, so I should just remove qtchooser from the build and packaging?

Reply:
Good question. If we have the tools in path there shouldn't be need to pull it in anymore. One thing I started pondering though is how common it would be to install qt packages and executing some command without starting a new shell. That wouldn't actually work with the path in the .sh file.

Let's take a step back. So tools, qtcore-devel and qmake subpackages pull now in the qtchooser and anything using those is free to choose either /usr/bin or /usr/lib/qt5/bin. Qtdeclarative tools are inconsistent in the sense that they don't pull in qtchooser and require the latter path at the moment.

Some options:
a) add the path here, adjust tests etc and hope the shell case doesn't matter too much. either with or without qtchooser removal.
b) add qtchooser first as dependency to  qtdeclarative tool packages. adjust tests to remove path (or use /usr/bin). get rid of qtchooser later.
c) adjust the binary path in qtbase, remove qtchooser in one go. tests and other falldown fixed separately.

Every option has some pros and cons. I don't like the qtchooser, but writing this down the b) could be an easy way to just get the tests working and allowing later adjustment without breakage.

Reply:
b) sounds good to me, I've a patch here: https://github.com/sailfishos/qtdeclarative/pull/1